src/cargo/core/compiler/custom_build.rs

```diff
@@ -584,7 +626,8 @@ pub fn build_map<'b, 'cfg>(cx: &mut Context<'b, 'cfg>, units: &[Unit<'b>]) -> Ca
         }
 
         {
-            let key = unit.pkg
+            let key = unit
+                .pkg
```

Comment:
This seems like an unrelated formatting change that shouldn't appear in the same commit.

Reply:
Yea, I prefer to use rustfmt to format, but master has diverged quite a bit from the canonical format.  I'll try to remove the unrelated changes.

I was thinking I might post a separate PR running rustfmt, but that touches 123 files which I think probably can't be reviewed properly.  I'm not sure how you guys want to handle that moving forward.  Each rust release brings more changes to rustfmt.  It makes things more difficult the more the master branch diverges.